<launch>

  <arg name="INPUT_IMAGE" />
  <arg name="MODEL" />
  <arg name="GPU" default="0" />

  <node name="selective_search"
        pkg="jsk_perception" type="selective_search.py">
    <remap from="~input" to="$(arg INPUT_IMAGE)" />
  </node>

  <node name="fast_rcnn"
        pkg="jsk_perception" type="fast_rcnn.py">
    <remap from="~input" to="$(arg INPUT_IMAGE)" />
    <remap from="~input/rect_array" to="selective_search/output" />
    <rosparam subst_value="true">
      model: $(arg MODEL)
      gpu: $(arg GPU)
    </rosparam>
  </node>

</launch>
